packages/trader/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -0,0 +1,202 @@
+                                        className='dc-input--no-placeholder'
+                                        label={localize('Second line of address (optional)')}
+                                        placeholder={localize('Second line of address')}
+                                    />
+                                    <InputField
+                                        name='address_city'
+                                        required
+                                        className='dc-input--no-placeholder'
+                                        label={localize('Town/City')}
+                                        placeholder={localize('Town/City')}
+                                    />
+                                    <Dropdown
+                                        id='address_state'
+                                        className='address_state-dropdown'
+                                        classNameDisplay='dc-input address_state-dropdown__display'
+                                        classNameDisplaySpan='address_state-dropdown__display__span'
+                                        classNameItems='address_state-dropdown__items'
+                                        classNameLabel='address_state-dropdown__label'
+                                        label={localize('State/Province (optional)')}
+                                        list={this.props.states_list}
+                                        name='address_state'
+                                        value={values.address_state}
+                                        onChange={handleChange}
+                                    />
+                                    <InputField
+                                        name='address_postcode'
+                                        required
+                                        className='dc-input--no-placeholder'
+                                        label={localize('Postal/ZIP Code')}
+                                        placeholder={localize('Postal/ZIP Code')}
+                                    />
+                                </div>
+                                <p className='personal-details-form__description'>
+                                    <Localize
+                                        i18n_default_text={'Any information you provide is confidential and will be used for verification purposes only.'}
+                                    />
+                                </p>
+                            </div>
+                            <FormSubmitButton
+                                is_disabled={
+                                    // eslint-disable-next-line no-unused-vars
+                                    isSubmitting ||
+                                    Object.keys(errors).length > 0
+                                }
+                                label='Next'
```

Comment:
localize?

Reply:
It is handled inside form submit button component.